web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -1844,16 +1839,89 @@ form#views-exposed-form-job-board-page {
 //Checkout Pages
 
 .layout-checkout-form {
+  background: $fmcg-black;
   box-shadow: none;
-  padding-left: 0;
-  padding-right: 0;
-  padding-bottom: 10px;
-  border-bottom: 0;
+
+  .checkout-pane {
+    padding-right: 24px;
+  }
+
+  a {
+    color: $fmcg-chrome;
+    font-size: 14px;
+    vertical-align: middle;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $fmcg-yellow;
+    }
+  }
+
+  .col-md-8, .col-xs-12 {
```

Comment:
Is there a way of dealing with this without altering the styles of the .col-* selectors?